internal/adapters/uiterm/terminal_test.go

```
@@ -0,0 +1,149 @@
+	}
+	if snapshot.Columns != 6 || snapshot.Rows != 3 || snapshot.Revision == 0 {
+		t.Fatalf("resize snapshot = %#v", snapshot)
+	}
+}
+
+func TestTerminalFailedTransactionMakesCaptureUnavailable(t *testing.T) {
+	terminal := newTestTerminal(t, 4, 2)
+	terminal.BeginOutput(ports.UIContext{})
+	_, _ = terminal.Write([]byte("data"))
+	terminal.EndOutput(false)
+	if _, err := terminal.Snapshot(); err != ports.ErrUIUnavailable {
```

golangci-lint (2.13.2)

[error] comparing with != will fail on wrapped errors. Use errors.Is to check for a specific error

(errorlint)